driver/pace/driver/diagnostics.py

```diff
@@ -7,11 +7,10 @@
 import pace.driver
 import pace.dsl
 import pace.stencils
-import pace.util
-import pace.util.grid
 from pace.dsl.dace.orchestration import dace_inhibitor
 from pace.fv3core.dycore_state import DycoreState
 from pace.util.constants import RGRAV
+from pace.util.grid.helper import GridData
```

Comment:
This is not going to work as there are still references to pace.util and pace.util.grid in the code

Reply:
If this needs to be refactored them we have to change calls to things like `pace.util.Z_DIM` on line 51